datawig/imputer.py

```diff
@@ -209,7 +210,17 @@ def fit(self,
         self.batch_size = batch_size
         self.final_fc_hidden_units = final_fc_hidden_units
 
-        self.ctx = ctx
+        gpu_list = []
```

Comment:
this looks like it would also fit nicely in the utils function instead of the imputer itself.

do you think it would be possible to simply extend the functionality of gpu_device to return a list?